modules/src/core/ics02_client/client_state.rs

```diff
@@ -40,6 +40,15 @@ pub trait ClientState: Clone + core::fmt::Debug + Send + Sync {
     /// Frozen height of the client
     fn frozen_height(&self) -> Option<Height>;
 
+    /// The client's unbonding period
+    fn unbonding_period(&self) -> Duration;
```

Comment:
I think `unbonding_period` is Tendermint (or more generally PoS) specific so should not be a part of this trait.

Reply:
agreed
I am also concerned about ``AnyClientState::trust_threshold`` for the same reason :thinking:

Reply:
Fixed in https://github.com/informalsystems/ibc-rs/pull/1625/commits/4b0a78dd082066a18cd11514af8e8b85ec5526f6